examples/gallery/plot/colorbar.py

```diff
@@ -0,0 +1,52 @@
+"""
+Colorbar
+--------
+
+The :meth:`pygmt.Figure.colorbar` method creates a color scalebar. We must specify the
+colormap via the ``cmap`` argument, and set the placement via the ``position`` argument.
+The full list of color paletted tables can be found at :gmt-docs:`cookbook/cpts.html`.
+You can set the `position` of the colorbar using the following options:
+
+- j/J: justified inside/outside the mapframe using any 2 character combination of
+  vertical (**T** op, **M** iddle, **B** ottom) and horizontal (**L** eft, **C** enter,
+  **R** ight) alignment codes, e.g. `position="jTR"` for top right.
+- g: using map coordinates, e.g. `position="g170/-45"` for longitude 170E, latitude 45S.
+- x: using paper coordinates, e.g. `position="x5c/7c"` for 5cm,7cm from anchor point.
+- n: using normalized (0-1) coordinates, e.g. `position="n0.4/0.8"`.
+
+Note that the anchor point defaults to the bottom left (BL). Append +h to ``position``
+to get a horizontal colorbar instead of a vertical one. For more advanced styling
+options, see the full option list at :gmt-docs:`colorbar.html`.
+"""
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 3, 6, 9], projection="t0/3c", frame=True)
+
+# Create a colorbar suitable for surface topography- oleron
+
+fig.colorbar(
+    cmap="oleron",
+    position="jTC+w6c/1c+h",  # justified inside map frame (j) at Top Center (TC)
+    box=True,
+    frame=["+Loleron", "xaf", "y+lm"],
```

Comment:
Have we documented how the argument
```python
frame=["+Loleron", "xaf", "y+lm"]
```
is equivalent to
```
-B+Loleron -Bxaf -By+lm
```
(i.e. use a list to replicate repeated flags) somewhere? I think it could be a good item to add to the [Note for experienced GMT users](https://www.pygmt.org/dev/tutorials/first-figure.html#note-for-experienced-gmt-users) in the docs and can do so if desired.

Reply:
Funny you said that, I only discovered that recently too at https://github.com/GenericMappingTools/pygmt/pull/325#issuecomment-540203439 but didn't know where best to put it in the docs.

> I think it could be a good item to add to the [Note for experienced GMT users](https://www.pygmt.org/dev/tutorials/first-figure.html#note-for-experienced-gmt-users) in the docs and can do so if desired.

Yep, go for it!